src/client/content_cao.cpp

```diff
@@ -810,6 +813,10 @@ void GenericCAO::addToScene(ITextureSource *tsrc, scene::ISceneManager *smgr)
 	if (node && m_matrixnode)
 		node->setParent(m_matrixnode);
 
+	ShadowRenderer *shadow = RenderingEngine::get_shadow_renderer();
```

Comment:
this assigment must be done in condition as shadow is not used after

Reply:
fixed